src/_pytest/python.py

```diff
@@ -1047,7 +1067,7 @@ def setmulti(
         *,
         argnames: Iterable[str],
         valset: Iterable[object],
-        id: str,
+        id: str | None,
```

Comment:
Why can't the same sentinel be reused in other places? Seeing `_HiddenParam` in some places but `None` in others is rather confusing. Isn't the purpose/semantics the same?